playgrounds/remix/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "remix-playground",
+  "private": true,
+  "sideEffects": false,
+  "version": "0.1.0",
+  "type": "module",
+  "scripts": {
+    "build": "remix vite:build",
+    "dev": "remix vite:dev",
+    "start": "remix-serve ./build/server/index.js",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@itwin/itwinui-react": "*",
+    "@remix-run/node": "^2.7.2",
+    "@remix-run/react": "^2.7.2",
+    "@remix-run/serve": "^2.7.2",
+    "isbot": "^4.1.0",
+    "react": "^18",
+    "react-dom": "^18"
+  },
+  "devDependencies": {
+    "@remix-run/dev": "^2.7.2",
```

Comment:
You can also bump all of these from `2.7.2` to `2.8.0`

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1887/commits/0506f39f64bdfa47817228642cc4731086e163ed)